angular-portfolio/src/app/research/research.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { ResearchService } from '../research.service';
+import { Res } from '../res-template';
+
+@Component({
+  selector: 'app-research',
+  templateUrl: './research.component.html',
+  styleUrls: ['./research.component.scss']
+})
+export class ResearchComponent implements OnInit {
+
+  research: Res[];
+
+  constructor(private researchService: ResearchService) {
+    this.getResearch();
+  }
+
+  ngOnInit(): void {
```

Comment:
It looks like you are not using this method. Please remove the method from the class. Also remove "implements OnInit" on the "export class" line